diff --git a/node_modules/draft-convert/esm/convertToHTML.js b/node_modules/draft-convert/esm/convertToHTML.js
index e764146..15c59d3 100644
--- a/node_modules/draft-convert/esm/convertToHTML.js
+++ b/node_modules/draft-convert/esm/convertToHTML.js
@@ -1,7 +1,7 @@
 // import Immutable from 'immutable'; // eslint-disable-line no-unused-vars
 import invariant from 'invariant';
 import React from 'react';
-import ReactDOMServer from 'react-dom/server';
+import ReactDOMServer from 'react-dom/server.browser';
 import { convertToRaw } from 'draft-js';
 import encodeBlock from './encodeBlock';
 import blockEntities from './blockEntities';
diff --git a/node_modules/draft-convert/esm/util/getBlockTags.js b/node_modules/draft-convert/esm/util/getBlockTags.js
index 853d80e..fa260aa 100644
--- a/node_modules/draft-convert/esm/util/getBlockTags.js
+++ b/node_modules/draft-convert/esm/util/getBlockTags.js
@@ -1,6 +1,6 @@
 import invariant from 'invariant';
 import React from 'react';
-import ReactDOMServer from 'react-dom/server';
+import ReactDOMServer from 'react-dom/server.browser';
 import splitReactElement from './splitReactElement';
 
 function hasChildren(element) {
diff --git a/node_modules/draft-convert/esm/util/getElementHTML.js b/node_modules/draft-convert/esm/util/getElementHTML.js
index dcec256..7549185 100644
--- a/node_modules/draft-convert/esm/util/getElementHTML.js
+++ b/node_modules/draft-convert/esm/util/getElementHTML.js
@@ -1,7 +1,7 @@
 import _typeof from "@babel/runtime/helpers/esm/typeof";
 import invariant from 'invariant';
 import React from 'react';
-import ReactDOMServer from 'react-dom/server';
+import ReactDOMServer from 'react-dom/server.browser';
 import splitReactElement from './splitReactElement';
 
 function hasChildren(element) {
diff --git a/node_modules/draft-convert/esm/util/splitReactElement.js b/node_modules/draft-convert/esm/util/splitReactElement.js
index 589732c..3f64602 100644
--- a/node_modules/draft-convert/esm/util/splitReactElement.js
+++ b/node_modules/draft-convert/esm/util/splitReactElement.js
@@ -1,6 +1,6 @@
 import invariant from 'invariant';
 import React from 'react';
-import ReactDOMServer from 'react-dom/server'; // see http://w3c.github.io/html/syntax.html#writing-html-documents-elements
+import ReactDOMServer from 'react-dom/server.browser'; // see http://w3c.github.io/html/syntax.html#writing-html-documents-elements
 
 var VOID_TAGS = ['area', 'base', 'br', 'col', 'embed', 'hr', 'img', 'input', 'link', 'meta', 'param', 'source', 'track', 'wbr'];
 export default function splitReactElement(element) {
