diff --git a/node_modules/svelte/internal/index.mjs b/node_modules/svelte/internal/index.mjs
index 828cbc4..590e730 100644
--- a/node_modules/svelte/internal/index.mjs
+++ b/node_modules/svelte/internal/index.mjs
@@ -1728,6 +1728,7 @@ function debug(file, line, column, values) {
 let on_destroy;
 function create_ssr_component(fn) {
     function $$render(result, props, bindings, slots, context) {
+        props.$$slots = slots
         const parent_component = current_component;
         const $$ = {
             on_destroy,
