diff --git a/node_modules/svelte/internal/index.mjs b/node_modules/svelte/internal/index.mjs
index 02d1f27..d6c01f5 100644
--- a/node_modules/svelte/internal/index.mjs
+++ b/node_modules/svelte/internal/index.mjs
@@ -345,6 +345,7 @@ function add_resize_listener(element, fn) {
     }
     const object = document.createElement('object');
     object.setAttribute('style', 'display: block; position: absolute; top: 0; left: 0; height: 100%; width: 100%; overflow: hidden; pointer-events: none; z-index: -1;');
+    object.setAttribute('aria-label', 'resizer');
     object.type = 'text/html';
     object.tabIndex = -1;
     let win;
@@ -1212,6 +1213,7 @@ function debug(file, line, column, values) {
 let on_destroy;
 function create_ssr_component(fn) {
     function $$render(result, props, bindings, slots) {
+        props.$$slots = slots
         const parent_component = current_component;
         const $$ = {
             on_destroy,
