diff --git a/node_modules/svelte/internal/index.mjs b/node_modules/svelte/internal/index.mjs
index 2afeff9..c54c7cf 100644
--- a/node_modules/svelte/internal/index.mjs
+++ b/node_modules/svelte/internal/index.mjs
@@ -129,7 +129,9 @@ function insert(target, node, anchor) {
     target.insertBefore(node, anchor || null);
 }
 function detach(node) {
+  if(node.parentNode){
     node.parentNode.removeChild(node);
+  }
 }
 function detach_between(before, after) {
     while (before.nextSibling && before.nextSibling !== after) {
