diff --git a/node_modules/svelte-preprocess/src/transformers/pug.js b/node_modules/svelte-preprocess/src/transformers/pug.js
index c07a697..d6db8ca 100644
--- a/node_modules/svelte-preprocess/src/transformers/pug.js
+++ b/node_modules/svelte-preprocess/src/transformers/pug.js
@@ -12,7 +12,7 @@ mixin else
   block
 
 mixin elseif(condition)
-  | {:elseif !{condition}}
+  | {:else if !{condition}}
   block
 
 mixin each(loop)
